README.md

```diff
@@ -86,6 +86,49 @@ const SomeComponent = () => {
 };
 ```
 
+## Additional features
+
+### Caching
+
+There is the option to invalidate the cache, meaning next time the hook is called it will fetch the data again.
+
+By setting a timeTillCacheInvalidate time in ms, as follows:
+
+**In `SomeHighLevelComponent.tsx`**
+
+```tsx
+import React from 'react';
+import { Provider } from 'react-redux';
+import { store } from './redux/store';
+import { State, Thing } from './types';
```

Comment:
Do we need this import here for the code example? It's not being used in the code